plugin-templates/src/features/class-search-customizations.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Search_Customizations class file
+ *
+ * @package create-wordpress-plugin
+ */
+
+namespace Create_WordPress_Plugin\Features;
+use function Create_WordPress_Plugin\register_meta_helper;
+
+use Alley\WP\Types\Feature;
+
+final class Search_Customizations implements Feature {
+	/**
+	 * Boot the feature.
+	 */
+	public function boot(): void {
+		add_action( 'elasticsearch_extensions_config', [ $this, 'elasticsearch_extensions_config' ] );
+	}
+
+	/**
+	 * Configure Elasticsearch Extensions.
+	 */
+	public function elasticsearch_extensions_config( $es_config ) {
+		$es_config->enable_empty_search()
+			->enable_post_type_aggregation()
+			->enable_taxonomy_aggregation( 'category' )
+			->restrict_post_types( [ 'post', 'page' ] );
```

Comment:
it could be kind of cool to move these values to use dependency injection, so that we can just pass them when the feature is registered. Like I did [here](https://github.com/alleyinteractive/create-wordpress-project/blob/develop/plugin-templates/features.txt#L5) and [here](https://github.com/alleyinteractive/create-wordpress-project/blob/develop/plugin-templates/src/features/class-primary-term-rest.php#L20)

That is definitely not blocking and can be changed later... just throwing it out there as the direction we're heading with things.

Reply:
good idea!